memtable/write_buffer_manager.cc

```diff
@@ -173,9 +179,8 @@ void WriteBufferManager::RemoveDBFromQueue(StallInterface* wbm_stall) {
 
   // Deallocate the removed nodes outside of the lock.
   std::list<StallInterface*> cleanup;
-
-  if (enabled() && allow_stall_) {
```

Comment:
Optimization 2: it appears to me that we shouldn't check `enabled() && allow_stall_` again for the following reasons:
- allow_stall_  can be changed to false in runtime and we don't want it to interfere with RemoveDBFromQueue() as a DB cleanup
- Even when allow_stall_ was not run-time changeable before, the `queue_` wouldn't have anything if `enabled() && allow_stall_` was false cuz we won't have the stall on then.